Source/Mock.cs

```diff
@@ -768,7 +768,7 @@ private static Expression GetPropertyExpression(Type mockType, PropertyInfo prop
 		/// Gets the interceptor target for the given expression and root mock, 
 		/// building the intermediate hierarchy of mock objects if necessary.
 		/// </summary>
-		private static Interceptor GetInterceptor(Expression fluentExpression, Mock mock)
+		internal static Interceptor GetInterceptor(Expression fluentExpression, Mock mock)
```

Comment:
Making this and the following helper methods `internal` was a quick hack to see if the reimplementation was feasible at all. This needs to be cleaned up. Do these methods actually have to be located in the `Mock` class?